src/routes/api/users/userRoutes.js

```diff
@@ -8,6 +8,8 @@ import Social from '../../../controllers/socialAuth';
 const router = express.Router();
 router.post('/signup', validateUserData.createUser, validateUserData.verifyIfEmailisAvailable ,UserController.signUp);
```

Comment:
I think here you can use one "router." instead of repeating it several times and the put the semicolon on the last statement.

Reply:
@Edmondus i have two routes, one for forget password and other for reset password. and for for the semicolon i fix that too.thanks